warehouse/query-core/src/main/java/datawave/query/iterator/logic/IndexIterator.java

```diff
@@ -137,8 +137,7 @@ private IndexIterator(Text field, Text value, SortedKeyValueIterator<Key,Value>
         
         this.source = source;
         // wrap the source with a limit
-        limitedSource = new LimitedSortedKeyValueIterator(source);
-        this.timeFilter = timeFilter;
+        this.limitedSource = new LimitedSortedKeyValueIterator(source);
```

Comment:
Consider making the `limitedSource` field `final`

Reply:
done